### ISSUE TITLE
php8.0.8下安装easywechat6.0找不到相应的版本

### ISSUE THREAD

**Issue by alawnchen** [w7corp/easywechat#2174]
## 我用的环境

* PHP 版本：8.0.8
* 是否使用了框架？框架名称：hyperf2.2

## 问题及现象

composer安装easywechat6.0的时候报错如下：

Could not find package w7corp/easywechat in a version matching "^6.0" and a stabili  
  ty matching "dev". 

我去查了packagist.org，确实没有看到有6.0的版本号。有点奇怪

**Comment by overtrue** [comment]
6.0 还没正式发布哈，可以这样子尝鲜：

```
composer require w7corp/easywechat:dev-6.x
```

**Comment by alawnchen** [comment]
嗯嗯，谢谢

**Comment by overtrue** [comment]
客气了

**Comment by shijunti19** [comment]
@overtrue 现在应该怎么安装，都试过了，都提示未找到
composer why symfony/event-dispatcher
symfony/http-kernel  v5.4.2  requires  symfony/event-dispatcher (^5.0|^6.0) 

[InvalidArgumentException]
  Could not find package w7corp/easywechat in a version matching dev-develop

**Comment by overtrue** [comment]
```bash
$ composer require w7corp/easywechat:dev-master
// or
$ composer require w7corp/easywechat:^6.0-beta13
```

**Comment by shijunti19** [comment]
@overtrue 
tories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires w7corp/easywechat dev-master -> satisfiable by w7corp/easywechat[dev-master].
    - w7corp/easywechat dev-master requires symfony/http-foundation ^6.0 -> found symfony/http-foundation[v6.0.0-BETA1, ..., 6.1
.x-dev] but the package is fixed to v5.4.2 (lock file version) by a partial update and that version does not match. Make sure yo
u list it as an argument for the update command.

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific
 versions.

这个怎么解决，我现在急需在laravel8上安装，我全新创建了，还是提示这个
